### ISSUE TITLE
Docker port are open on the wrong container

### ISSUE THREAD

**Issue by cesartl** [telepresenceio/telepresence#671]
When using `--docker-run` with docker port options `-p` the ports are open on the telepresence container rather than the user container:

![image](https://user-images.githubusercontent.com/18382796/40770566-f0c00a34-64b2-11e8-9c53-ad736eb919cb.png)

**Comment by cesartl** [comment]
Any update on this? This is critical for us as it prevents local debugging

**Comment by ark3** [comment]
Sorry for the lack of updates. Can you please help me understand the problem? What are you trying to do that doesn't work correctly?

It is true that `docker ps`/`docker port` show the published ports on the Telepresence network container rather than on the user container. However, the two containers share the same network. At least on my available test systems (Mac and Ubuntu, Docker 18.06), I see that connecting to a published port reaches the code running in my user container.

First terminal:

```shell
$ telepresence --docker-run --rm -it datawire/qotm:1.3 -p 5000:5000

T: Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.
2018-08-01 20:09:40 QotM 1.3 INFO: initializing on 434a015cc8a0:5000
2018-08-01 20:09:40 QotM 1.3 INFO:  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
2018-08-01 20:09:40 QotM 1.3 INFO:  * Restarting with stat
2018-08-01 20:09:40 QotM 1.3 INFO: initializing on 434a015cc8a0:5000
2018-08-01 20:09:40 QotM 1.3 WARNING:  * Debugger is active!
2018-08-01 20:09:40 QotM 1.3 INFO:  * Debugger PIN: 125-975-564
```

Second terminal:

```shell
$ docker ps
CONTAINER ID        IMAGE                              COMMAND                  CREATED             STATUS             
PORTS                    NAMES
3c3e7cf0626e        datawire/qotm:1.3                  "python3 qotm/qotm.py"   39 seconds ago      Up 38 seconds      
                         telepresence-1533154178-853344-71538                                                          
434a015cc8a0        datawire/telepresence-local:0.90   "/sbin/tini -v -- py…"   44 seconds ago      Up 42 seconds      
0.0.0.0:5000->5000/tcp   telepresence-1533154174-56162-71538                                                           
$ # port is on Tel container, not qotm container, but it'll still work
$ curl localhost:5000
{
  "hostname": "434a015cc8a0", 
  "ok": true, 
  "quote": "Non-locality is the driver of truth. By summoning, we vibrate.", 
  "time": "2018-08-01T20:10:48.918464", 
  "version": "1.3"
}
```

And back on the first terminal, we can see the log lines associated with that `curl`:

```shell
2018-08-01 20:10:48 QotM 1.3 DEBUG: GET /: session None, username None, handler statement                              
2018-08-01 20:10:48 QotM 1.3 INFO: 172.17.0.1 - - [01/Aug/2018 20:10:48] "GET / HTTP/1.1" 200 -                        
```

**Comment by cesartl** [comment]
Yes you are right it works as expected. It must have been something else which caused the issue and seeing the change I thought it was the cause
